configure

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+# 
+# Attribution Notice
+# ------------------
+# This file uses parts of Node.js `configure`.
+# Please refer to https://github.com/joyent/node.
+#
+import optparse
+import os
+import pprint
+import re
+import shlex
+import subprocess
+import sys
+import platform
+
+CC = os.environ.get('CC', 'cc')
+
+root_dir = os.path.dirname(__file__)
+build_dir = './build'
+
+# Parse options
+parser = optparse.OptionParser()
+parser.add_option("--debug",
+    action="store_true",
+    dest="debug",
+    help="Also build the debug build.")
+
+parser.add_option("--dest-cpu",
+    action="store",
+    dest="dest_cpu",
+    help="CPU architecture to build for. Valid values are: ia32, x64.")
+
+#parser.add_option("--shared",
+#    action="store_true",
+#    dest="shared",
+#    help="Build and use shared libsnowcrash instead of static one.")
```

Comment:
I don't know the purpose of this commented out part. Can we remove it completely if it is not going to be used? /cc @zdne